package.json

```diff
@@ -87,5 +87,13 @@
       "vue-cli-service lint",
       "git add"
     ]
+  },
+  "prettier": {
```

Comment:
I don't think we've prettier setup in this project. Do you intend to do so? I would definitely recommend to do in another PR as it might update a lot of files all at once.

Reply:
👍